.buildkite/pipeline.yml

```diff
@@ -1,5 +1,44 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+env:
+  SETUP_GOLANG_VERSION: '1.18.10'
```

Comment:
It's a pitty there is no `.go-version` file in this repository.

Reply:
means to add it?

Reply:
No, if they are not using it. Let's keep it as it was set in Jenkins ([using an env. variable](https://github.com/elastic/elastic-agent-client/blob/aa961fb53dddc0aadff1858582558967222eecb6/.ci/Jenkinsfile#L10)).